ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -46,6 +46,8 @@ singleExpression
 
 statement
     : query                                                                 #queryStatement
+    | expression                                                            #expressionEvaluate
+    | EVALUTE expression                                                    #evaluateExpression
```

Comment:
what's the different with calling `expression` and `EVALUATE expression`? Isn't the same, both calling the expression?

Btw, all DBs I've seen use the select clause for calling an expression, i.e. `select expression;`. Can we do it like that? We could call `select expr1, expr2, expr3;` and display one row in 3 columns with one expression each.

I think you can make all clauses in the select optional, like `query: SELECT selectItem (',' selectItem)* (<the rest>)?`